Serene/Serene.Web/Modules/BasicSamples/Dialogs/EntityDialogAsPanel/Index.cshtml

```diff
@@ -36,9 +36,9 @@
         myDialogAsPanel = new Serene.BasicSamples.EntityDialogAsPanel();
         // load a new entity if url doesn't contain an ID, or load order with ID specified in page URL
         // here we use done event in second parameter, to be sure operation succeeded before showing the panel
-        myDialogAsPanel.load(@Html.Raw(Model.ToJson()) || {}, () => {
+        myDialogAsPanel.load(@Html.Raw(Model.ToJson()) || {}, function () => {

```

Comment:
Blarg...I have no access to git outside of the web interface at my work, so all edits are done (poorly) by hand.